DP3TApp/Screens/Homescreen/BegegnungenDetail/NSBegegnungenDetailViewController.swift

```diff
@@ -49,9 +46,7 @@ class NSBegegnungenDetailViewController: NSTitleViewScrollViewController {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         navigationController?.setNavigationBarHidden(false, animated: true)
-        #if ENABLE_SYNC_LOGGING
-            lastSyncronizationControl.lastSyncronizationDate = NSSynchronizationPersistence.shared?.fetchLatestSuccessfulSync()?.date
-        #endif
+        lastSyncronizationControl.lastSyncronizationDate = NSSynchronizationPersistence.shared?.fetchLatestSuccessfulSync()?.date
```

Comment:
We should also refresh the date when the app comes from background (and the user is already on this view)